src/redist/targets/BundledTemplates.targets

```diff
@@ -20,94 +20,93 @@
-    <Bundled21Templates Include="Microsoft.DotNet.Web.ItemTemplates" PackageVersion="$(AspNetCorePackageVersionFor21Templates)" />
-    <Bundled21Templates Include="Microsoft.DotNet.Web.ProjectTemplates.2.1" PackageVersion="$(AspNetCorePackageVersionFor21Templates)" UseVersionForTemplateInstallPath="true" />
-    <Bundled21Templates Include="Microsoft.DotNet.Web.Spa.ProjectTemplates.2.1" PackageVersion="$(AspNetCorePackageVersionFor21Templates)" />
-    <Bundled21Templates Include="NUnit3.DotNetNew.Template" PackageVersion="$(NUnit3Templates21PackageVersion)" />
-  </ItemGroup>
+    <ItemGroup>
+      <CurrentVersionBundledTemplates Include="@(Bundled60Templates)" TemplateFrameworkVersion="6.0"/>
+    </ItemGroup>
 
-  <ItemGroup>
-    <CurrentVersionBundledTemplates Include="@(Bundled60Templates)" TemplateFrameworkVersion="6.0"/>
-  </ItemGroup>
+    <ItemGroup>
+      <PreviousVersionBundledTemplates Include="@(Bundled50Templates)" TemplateFrameworkVersion="5.0"/>
+      <PreviousVersionBundledTemplates Include="@(Bundled31Templates)" TemplateFrameworkVersion="3.1"/>
+      <PreviousVersionBundledTemplates Include="@(Bundled30Templates)" TemplateFrameworkVersion="3.0"/>
+      <PreviousVersionBundledTemplates Include="@(Bundled21Templates)" TemplateFrameworkVersion="2.1"/>
+    </ItemGroup>
 
-  <ItemGroup>
-    <PreviousVersionBundledTemplates Include="@(Bundled50Templates)" TemplateFrameworkVersion="5.0"/>
-    <PreviousVersionBundledTemplates Include="@(Bundled31Templates)" TemplateFrameworkVersion="3.1"/>
-    <PreviousVersionBundledTemplates Include="@(Bundled30Templates)" TemplateFrameworkVersion="3.0"/>
-    <PreviousVersionBundledTemplates Include="@(Bundled21Templates)" TemplateFrameworkVersion="2.1"/>
-  </ItemGroup>
+    <ItemGroup>
+      <BundledTemplates Include="@(CurrentVersionBundledTemplates)" />
+      <BundledTemplates Include="@(PreviousVersionBundledTemplates)" Condition="$(ProductMonikerRid.StartsWith('win'))" />
+    </ItemGroup>
 
-  <ItemGroup>
-    <BundledTemplates Include="@(CurrentVersionBundledTemplates)" />
-    <BundledTemplates Include="@(PreviousVersionBundledTemplates)" Condition="$(ProductMonikerRid.StartsWith('win'))" />
-  </ItemGroup>
+    <ItemGroup>
+      <BundledTemplates Update="@(BundledTemplates)">
+        <RestoredNupkgPath>$(NuGetPackageRoot)$([System.String]::copy('%(Identity)/%(PackageVersion)/%(Identity).%(PackageVersion).nupkg').ToLower())</RestoredNupkgPath>
+      </BundledTemplates>
+    </ItemGroup>
 
-  <ItemGroup>
-    <BundledTemplates Update="@(BundledTemplates)">
-      <NupkgPathRelativeToPackageRoot>%(Identity)/%(PackageVersion)/%(Identity).%(PackageVersion).nupkg</NupkgPathRelativeToPackageRoot>
-      <RestoredNupkgPath>$(NuGetPackageRoot)$([MSBuild]::ValueOrDefault('%(NupkgPathRelativeToPackageRoot)', '').ToLower())</RestoredNupkgPath>
```

Comment:
Moving the ItemGroup inside the target caused the `%(NupkgPathRelativeToPackageRoot)` reference to break with the following error.

```
D:\repos\installer\src\redist\targets\BundledTemplates.targets(104,5): error MSB4096: The item "Microsoft.DotNet.Common.ItemTemplates" in item list "BundledTemplates" does not define a value for metadata "NupkgPathRelativeToPackageRoot".  In order to use this metadata, either qualify it by specifying %(BundledTemplates.NupkgPathRelativeToPackageRoot), or ensure that all items in this list define a value for this metadata. [D:\repos\installer\src\redist\redist.csproj]
```

I solved this by in-lining the property which appears to only be referenced in this one spot.